loopy/target/c/__init__.py

```diff
@@ -1264,7 +1304,19 @@ class CTarget(CFamilyTarget):
     """This target may emit code using all features of C99.
     For a target base supporting "least-common-denominator" C,
     see :class:`CFamilyTarget`.
+
+    .. attribute:: has_gnu_libc
+
+        An instance of :class:`bool` indicating whether the kernel will be
+        compiled with C-compiler that's built with GNU-libc as its standard
+        C-library.
     """
+    hash_fields = CFamilyTarget.hash_fields + ("has_gnu_libc",)
+    comparison_fields = CFamilyTarget.comparison_fields + ("has_gnu_libc",)
+
+    def __init__(self, fortran_abi=False, has_gnu_libc=False):
+        super().__init__(fortran_abi=fortran_abi)
```

Comment:
Maybe you want a check like the following here too.
```
        if not isinstance(has_gnu_libc, bool):
            raise ValueError("has_gnu_libc must be a bool instance,"
                             f" got {has_gnu_libc}.")
```

Reply:
Thanks, done!